elements/document/collection/nuxeo-collection-create-layout.html

```diff
@@ -33,6 +33,14 @@
       required
     >
     </nuxeo-input>
+    <nuxeo-input
```

Comment:
Usually we leave a space between widgets.
```suggestion

    <nuxeo-input
```